packages/shorebird_cli/test/src/commands/apps/list_apps_command_test.dart

```diff
@@ -84,12 +84,47 @@ void main() {
 ┌───────────────────┬──────────────────────────────────────┬─────────┬───────┐
 │ Name              │ ID                                   │ Release │ Patch │
 ├───────────────────┼──────────────────────────────────────┼─────────┼───────┤
-│ Shorebird Counter │ 30370f27-dbf1-4673-8b20-fb096e38dffa │ 1.0.0   │ 1     │
-├───────────────────┼──────────────────────────────────────┼─────────┼───────┤
 │ Shorebird Clock   │ 05b45471-a5f3-48cd-b26a-da29d95914a7 │ --      │ --    │
+├───────────────────┼──────────────────────────────────────┼─────────┼───────┤
+│ Shorebird Counter │ 30370f27-dbf1-4673-8b20-fb096e38dffa │ 1.0.0   │ 1     │
```

Comment:
FYI: Needed to switch the order of the output because `Shorebird Clock` becomes before `Shorebird Counter` when sorting alphabetically.